web/styles/popovers.css

```diff
@@ -1098,20 +1126,36 @@ ul {
     }
 }
 
-.personal-menu-actions {
+.navbar-dropdown-menu {
     display: flex;
     flex-flow: column nowrap;
+    margin: 0;
+    padding: 0;
+    border: solid 1px var(--color-border-dropdown-menu);
+    background-color: var(--color-background-dropdown-menu);
+    max-height: 85vh;
+    min-width: 230px;
+    overflow-x: hidden;
+    user-select: none;
+    border-radius: 6px;
+    box-shadow: var(--box-shadow-navbar-dropdown-menu);
 
-    .list {
+    .simplebar-content {
+        min-width: max-content;
+    }
+
+    & ul {
```

Comment:
I don't see how this selector is ever reached. That said, the HTML in `gear_menu_popover.hbs` is kind of a mess. `<div>` elements should not be children of `<ul>` elements, so it's possible that this selector will be needed eventually. But I'll remark more about that below.

Reply:
Modified it use the new structure.